pkg/core/xds/matched_policies.go

```diff
@@ -27,7 +27,8 @@ type MatchedPolicies struct {
 	TrafficRoutes      RouteMap
 
 	// Dataplane -> Policy
-	TrafficTrace *core_mesh.TrafficTraceResource
+	TrafficTrace  *core_mesh.TrafficTraceResource
+	ProxyTemplate *core_mesh.ProxyTemplateResource
```

Comment:
Please drop a comment that actual Envoy configuration is generated without taking into account this `ProxyTemplate` field (and the same for `TrafficRoutes` field).